### PR TITLE
Module 1 Capstone Project - Jazz Concert

### PR DESCRIPTION
**Pull request by binyamolango**
# Hello Reviewer 👋

> Here are the changes I implemented to the project

- Followed these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
- Personalized the content of my page differently than the one in the original design.
- Built the home page and the About page that each have versions for 2 different screen sizes:
 Mobile: up to 768px wide and Desktop: 768px or wider.
- Added a menu to the mobile version that has a link to the About page.
- Implemented the menu to work as when the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page and has a close (X) button that closes the menu.
- Created the section "Featured musicians" dynamically in JavaScript.

[Here is a link to loom walkthrough about the project](https://www.loom.com/share/14cfb3f95e504aeabb669072f40aa18a?sid=0bf830cc-a394-46fe-980e-d9a0801741fc)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -16,8 +16,14 @@
         <div class="header-top">
             <div class="container row">
```
Comment (zieeco) [inline about.html:17]: - Hey, to enforce good use of the semantic tag, kindly change this div to nav tag as the div is mainly used to divide different parts of the webpage and as such, it has no semantic meaning. the nav tag would be the correct tag to use here as this contains the introductory content of your page which in your case is best applicable to be used here. please, update this tag to improve your semantic usage. :pray:

File: style.css [hunk 2]
```diff
@@ -679,7 +721,7 @@ a {
   }
```
Comment (zieeco) [inline style.css:723]: ## partner logo

```

.partner-logo {
  opacity: 0.5;
}
```

- Hey, let's kindly add the `filter: grayscale(100%) contrast(30%);` to the partner images to make it have the gray color to make the color more like that on the main design to improve our design.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,252 @@
...
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Personal-Portfolio] <a name="about-project"></a>
+
```
Comment (zieeco) [inline README.md]: - Kindly update the name of this project to the current one as this. The name given is different from the project name.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @binyamolango,






Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Nice with your implementation :+1: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @binyamolango , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Well written README ✔️ 
- [ ] All Linter checks are  passing ✔️ 
- [ ]  All requested changes have been made  ✔️ 
- [ ] Project up and running ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**